[IMP] Inventory: making targeted edits for 16 (#4381)
odoo/documentation: content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst

   :align: center
   :alt: Activate multi-step routes and storage locations in Inventory settings.

> Next, the warehouse needs to be configured for three-step receipts.To do that, go to

Reviewer: Missing a space
```suggestion
Next, the warehouse needs to be configured for three-step receipts. To do that, go to
```

> :menuselection:`Inventory app --> Configuration --> Warehouses`, and select the desired warehouse to

Reviewer: ```suggestion
:menuselection:`Inventory app --> Configuration --> Warehouses`, and select the desired :guilabel:`warehouse` to
```

> Next, the warehouse needs to be configured for three-step receipts. Go to :menuselection:`Inventory
> --> Configuration --> Warehouses`, and click :guilabel:`Edit` to edit the warehouse. Then, select
> :guilabel:`Receive goods in input, then quality and then stock (3 steps)` for :guilabel:`Incoming

Reviewer 2: ```suggestion
Next, the warehouse needs to be configured for three-step receipts. To do that, go to
:menuselection:`Inventory app --> Configuration --> Warehouses`, and select the desired warehouse
to be edited. Doing so reveals the detail form for that specific warehouse.

On that warehouse detail form page, select :guilabel:`Receive goods in input, then quality and then
stock (3 steps)` for :guilabel:`Incoming
```

added specificity - broke up chunky paragraph into 2 smaller ones

Shipments`.

.. image:: receipts_three_steps/three-step-receipt-settings.png
   :align: center
   :alt: Set incoming shipment option to receive in three steps.

> Activating three-step receipts and deliveries will create two new internal locations:
> *Input* (WH/Input), and *Quality Control* (WH/Quality Control). To rename these locations, go to

Reviewer 2: ```suggestion
Activating three-step receipts and deliveries creates two new internal locations: *Input*
(WH/Input), and *Quality Control* (WH/Quality Control). To rename these locations, go to
```
removed "will" - establish present tense

> :menuselection:`Configuration --> Locations`, click on the :guilabel:`Location` to change, and
> update the name.

Reviewer 2: ```suggestion
:menuselection:`Inventory app --> Configuration --> Locations`, then click on the
desired :guilabel:`Location` to change (or update) the name.
```

> Process a receipt in three steps (input + quality + stock)

Reviewer 2: ```suggestion
Receipts in three steps (input, quality, and stock)
```
removed "action"/verb from heading - removed "+"

==========================================================

> 

Reviewer 2: Could be nice to add a quick 1-2 sentence intro into the section that follows - just a suggestion

Author: This is also explained in the very beginning why this would be beneficial. I could move it, but I wanted some kind of introduction at the top, and didn't want to say it twice.
